src/ToggleFunction.tsx

```diff
@@ -1,13 +1,18 @@
-export const getFavFromLocalStorage = () => {
-  const favourites = JSON.parse(localStorage.getItem("favourites")) || [];
+import { Photo } from "./types";
+
+export const getFavFromLocalStorage = (): Photo[] => {
```

Comment:
Why do we need to use type here since there are no props?

`export const getFavFromLocalStorage = () => {`